core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -423,13 +430,16 @@ export class PickerColumnInternal implements ComponentInterface {
      * the attribute can be moved to datetime.tsx and set on every
      * instance of ion-picker-column-internal there instead.
      */
+
     return (
       <Host
         exportparts={`${PICKER_ITEM_PART}, ${PICKER_ITEM_ACTIVE_PART}`}
-        tabindex={0}
+        disabled={pickerDisabled}
+        tabindex={pickerDisabled ? null : 0}
         class={createColorClasses(color, {
           [mode]: true,
           ['picker-column-active']: isActive,
+          ['picker-column-disabled']: pickerDisabled,
```

Comment:
This class doesn't seem needed anymore. The only place it's used is in the CSS, and that could be swapped out for the `[disabled]` attribute.

Reply:
Sure. I went ahead and removed the `.picker-item-disabled` class while I was in there, by the same reasoning, even though it predates this PR.

Reply:
I was referring to `picker-column-disabled` not `picker-item-disabled`. The `picker-column-disabled` class was added in this PR.

Reply:
Re-read this and looks like I misunderstood your original comment. All of that sounds good!